db/schema.rb

```diff
@@ -10,21 +10,22 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2023_02_22_154316) do
+ActiveRecord::Schema[7.0].define(version: 2023_02_24_184326) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
   create_table "books", force: :cascade do |t|
     t.string "title"
     t.string "author"
     t.string "genre"
-    t.date "date_of_publication"
+    t.integer "date_of_publication"
```

Comment:
That does not work, please keep date of publication as a date :)

Reply:
check!